src/test/kotlin/Composite/CompositeTest.kt

```diff
@@ -0,0 +1,59 @@
+package Composite
+
+import Composite.base.ChineseCooker
+import Composite.base.Cooker
+import Composite.base.ItalianCooker
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+
+class CompositeTest {
+
+  @Test
+  fun `composite should allow to add more childrens`() {
+    val kitchen = MockKitchen()
+    assertThat(kitchen.cookers.size, `is`(0))
+
+    val italian = ItalianCooker()
+    kitchen.add(italian)
+    assertThat(kitchen.cookers.size, `is`(1))
+  }
+
+  @Test
+  fun `composite should allow to remove childrens`() {
```

Comment:
Seriously you are testing that you are implemented a good `MockKitchen` when calls kitchen.remove(italian) it calls list.remove